spec.bs

```diff
@@ -2032,14 +2028,16 @@ and a [=real time reporting contributions map=] |realTimeContributionsMap|:
          [=trusted bidding signals batcher/all per interest group data=].
       1. Let |fetchSignalDuration| be the [=duration from=] |fetchSignalStartTime| to |settings|'s
         [=environment settings object/current monotonic time=], in milliseconds.
-      1. If |perBuyerCumulativeTimeout| is not null, then decrement |perBuyerCumulativeTimeout| by
+      1. [=Expend cumulative timeout time=] given |cumulativeTimeoutTracker| and
         |fetchSignalDuration|.
       1. [=map/For each=] joiningOrigin → |groups| of |perSignalsUrlGenerator|:
         1. [=list/For each=] |ig| of |groups|:
-          1. If |perBuyerCumulativeTimeout| is negative, then [=iteration/break=].
+          1. If |cumulativeTimeoutTracker| [=cumulative timeout tracker/is expired=] then:

```

Comment:
nit: this can be merged as "if ... then increment ... and continue", to have less nest levels and steps

Reply:
Done